rl-header.dpatch by  <doko@debian.org>

Include stdio.h in readline.h and history.h

--- a/history.h
+++ b/history.h
@@ -32,6 +32,7 @@ extern "C" {
 #  include "rlstdc.h"
 #  include "rltypedefs.h"
 #else
+#  include <stdio.h>
 #  include <readline/rlstdc.h>
 #  include <readline/rltypedefs.h>
 #endif
--- a/readline.h
+++ b/readline.h
@@ -32,6 +32,7 @@ extern "C" {
 #  include "keymaps.h"
 #  include "tilde.h"
 #else
+#  include <stdio.h>
 #  include <readline/rlstdc.h>
 #  include <readline/rltypedefs.h>
 #  include <readline/keymaps.h>
